keyboards/mechwild/sugarglider/rules.mk

```diff
@@ -0,0 +1,24 @@
+# Build Options
+#   change yes to no to disable
+#
+WIDE_OLED_ENABLE = yes
```

Comment:
One of the screen sizes needs to be set or it will have issues with going out of bounds. I defaulted to wide because it was the most popular option purchased. Assigning it at the keymap level would not address the concerns about configurator making a firmware that doesn't produce a garbled OLED.

Reply:
I am refactoring this as nested hardware options with the blackpill type. This is not an elegant solution but it should allow it to work with configurator.